src/main/java/chess/domain/ChessBoard.java

```diff
@@ -0,0 +1,125 @@
+package chess.domain;
+
+import chess.domain.piece.*;
+import chess.domain.piece.info.Color;
+import chess.domain.piece.info.Direction;
+import chess.domain.piece.info.Position;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class ChessBoard {
+    private static final String OBSTACLE_ERROR = "[ERROR] 기물을 뛰어 넘어 이동할 수 없습니다.";
+    private static final String SAME_COLOR_ERROR = "[ERROR] taget에 같은 편 말이 있습니다.";
+    private static final String EMPTY_ERROR = "[ERROR] 체스말이 없습니다.";
+    private Map<Position, Piece> chessBoard;
+
+    public ChessBoard(Map<Position, Piece> chessBoard) {
+        this.chessBoard = chessBoard;
+    }
+
+    public static ChessBoard generate() {
+        Map<Position, Piece> chessBoard = new HashMap<>();
+        chessBoard.putAll(King.generate());
+        chessBoard.putAll(Queen.generate());
+        chessBoard.putAll(Knight.generate());
+        chessBoard.putAll(Rook.generate());
+        chessBoard.putAll(Bishop.generate());
+        chessBoard.putAll(Pawn.generate());
+        return new ChessBoard(chessBoard);
+    }
+
+    public Map<Position, Piece> getChessBoard() {
+        return chessBoard;
+    }
+
+    public Piece findByPosition(Position position) {
+        return chessBoard.keySet().stream()
+                .filter(key -> key.equals(position))
+                .map(key -> chessBoard.get(position))
+                .findFirst()
+                .orElse(Empty.EMPTY);
+    }
```

Comment:
```suggestion
    public Piece findByPosition(Position position) {
        return chessBoard.getOrDefault(position, Empty.EMPTY);
    }
```

스트림도 좋지만 map에서 이미 지원하는 것 같네요 :)